curl/Dockerfile

```diff
@@ -25,8 +28,9 @@ ARG DEFAULT_GROUPS="x25519:x448:kyber512:p256_kyber512:kyber768:p384_kyber768:ky
 ARG MAKE_DEFINES="-j 4"
 
 
-FROM alpine:3.11 as intermediate
+FROM alpine:3.20 as intermediate
```

Comment:
an Arg could also be used to define the base version (or whole image). For example
```
ARG BUILDVER=3.20
FROM alpine:$BUILDVER
```
Whilst this wouldn't change defaults for the user just taking the dockerfile to run, it would make it easier to build/test different variations in CI, or for testing against new versions

Reply:
I like this change, I'll make it